sim/main_game.py

```diff
@@ -1,12 +1,93 @@
 import sys
 import pygame
+import random
+import json
+from collections import defaultdict
 from settings.game_settings import GameSettings
 from characters.player.player import Player
 from characters.npc.npc import Npc
+from pathlib import Path
+
+image_folder_path = Path.cwd() / "images"
+settings_folder_path = Path.cwd() / "characters" / "npc" / "npc_settings.json"
+
+
+CHARACTER_NAMES = [
+    "npc1",
+    "npc2",
+    "npc3",
+    "npc4",
+    "npc5",
+    "npc6",
+    "npc7",
+    "npc8",
+    "npc9",
+    "npc10",
+    "npc11",
+    "npc12",
+    "npc13",
+    "npc14",
+]
+ATTRIBUTES = [
+    "Intelligence",
+    "Honesty",
+    "Patience",
+    "Humor",
+    "Ambition",
+    "Affection",
+    "Confidence",
+    "Adventurousness",
+    "Empathy",
+]
+COORDINATES = [
+    (0, 0),
+    (0, 256),
+    (0, 512),
+    (256, 0),
+    (256, 256),
+    (256, 512),
+    (512, 0),
+    (512, 256),
+    (768, 0),
+    (768, 256),
+    (768, 512),
+]
+PICKED_COORDINATES = []
+
+
+def generateNPC():
+    global PICKED_COORDINATES
```

Comment:
this might hint at using a class for better encapsulation, I would prevent mutating data in an outer scope, it can lead to unwanted side effects